src/redux/reducers/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import { searchFiltering, searchRetrieval } from './Browse';
+import { setLogin } from './User';
 
 export default combineReducers({
   searchFiltering,
   searchRetrieval,
+  setLogin,
```

Comment:
Actually combineReducers will allow using a different value for the key, to store the data under. So here it can be
```suggestion
  isLoggedIn: setLogin,
```